livekit-ffi/generate_proto.sh

```diff
@@ -20,6 +20,8 @@ OUT_RUST=src
 protoc \
     -I=$PROTOCOL \
     --prost_out=$OUT_RUST \
+    --prost_opt=compile_well_known_types \
+    --prost_opt=extern_path=.google.protobuf=::pbjson_types \
```

Comment:
Do we need json types? Seems like we never serialize to json?

Reply:
Not anymore, livekit-protocol was converting .google.protobuf types to pbjson_types but livekit-ffi was not doing so, it was messing up timestamps in metrics.proto. Since we have removed metrics proto from ffi, we don't need this anymore.